requests/yaf_request_simple.c

```diff
@@ -254,8 +254,10 @@ YAF_STARTUP_FUNCTION(request_simple){
 	yaf_request_simple_ce = zend_register_internal_class_ex(&ce, yaf_request_ce);
 	yaf_request_simple_ce->ce_flags |= ZEND_ACC_FINAL;
 
+#if PHP_VERSION_ID < 70100
 	zend_declare_class_constant_string(yaf_request_simple_ce, ZEND_STRL("SCHEME_HTTP"),  "http");
 	zend_declare_class_constant_string(yaf_request_simple_ce, ZEND_STRL("SCHEME_HTTPS"), "https");
+#endif
```

Comment:
Without this patch, build is ok, but a runtime error make it unusable:

      PHP Fatal error:  Cannot redefine class constant Yaf_Request_Simple::SCHEME_HTTP in Unknown on line 0

This error affects lot of extensions (imagick, gmagick...)